openstef_dbc/services/predictor.py

```diff
@@ -200,12 +200,15 @@ def get_load_profiles(
             "E4A_I",
         ]
 
-        field_selection = '" or r._field == "'.join(sjv_profles)
-
+        field_selection = '" or r["_field"] == "'.join(sjv_profles)
         query = f"""
-            from(bucket: "realised/autogen")
-                |> range(start: {bind_params["_start"].strftime('%Y-%m-%dT%H:%M:%SZ')}, stop: {bind_params["_stop"].strftime('%Y-%m-%dT%H:%M:%SZ')}) 
-                |> filter(fn: (r) => r._measurement == "sjv" and r._field == "{field_selection}")
+        from(bucket: "realised/autogen")
+            |> range(start: {bind_params["_start"].strftime('%Y-%m-%dT%H:%M:%SZ')}, stop: {bind_params["_stop"].strftime('%Y-%m-%dT%H:%M:%SZ')}) 
+            |> filter(fn: (r) => r["_measurement"] == "sjv")
+            |> filter(fn: (r) => r["_field"] == "{field_selection}")
+            |> aggregateWindow(every: {forecast_resolution[:-3]}m, fn: mean, createEmpty: false)
```

Comment:
Good to check if aggregation causes the last (or first) timestamp to disappear.

Reply:
Good point, I checked and added a shift, now both frames from the results from the old and the new queries are identical